jobsdb/migration.go

```diff
@@ -223,29 +274,53 @@ func (jd *HandleT) cleanupStatusTables(ctx context.Context, dsList []dataSetT) e
 				return err
 			}
 		}
+		for _, statusTable := range toVaccum {
```

Comment:
That shouldn't be possible because of the condition here: https://github.com/rudderlabs/rudder-server/blob/16b5c3429aa5bcc5d31178b22169b6546ca54c90/jobsdb/migration.go#L319